### PR TITLE
AlignAndFocusPowderFromFiles absorption test

### PR DESCRIPTION
**Pull request by peterfpeterson**
This adds a test to verify that the absorption workspace, when supplied, is applied correctly.

**To test:**

Look over the new test and see that the builds pass.

*There is no associated issue.*

*This does not require release notes* because it is adding a system test to #24821.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/analysis/AlignAndFocusPowderFromFilesTest.py [hunk 1]
```diff
@@ -7,15 +7,10 @@
...
 import os
-
-
-def _skip_test():
-    """Helper function to determine if we run the test"""
-    import platform
-
-    # Only runs on RHEL6 at the moment
-    return "Linux" not in platform.platform()
+import timing
```
Comment (marshallmcdonnell) [inline Testing/SystemTests/tests/analysis/AlignAndFocusPowderFromFilesTest.py]: I got `no module found` for `timing`. Add as dependency or switch to different module?

File: Testing/SystemTests/tests/analysis/AlignAndFocusPowderFromFilesTest.py [hunk 2]
```diff
@@ -190,3 +191,79 @@ def runTest(self):
...
+        PDDetermineCharacterizations(InputWorkspace=self.wksp_mem, Characterizations='characterizations',
+                                     ReductionProperties='__snspowderreduction_inner')
+
+        # calculate the absorption
+        num_wl_bins = 200
+        absorptionWS = WorkspaceFactory.create(mtd[self.wksp_mem],
+                                               NVectors=mtd[self.wksp_mem].getNumberHistograms(), XLength=num_wl_bins+1,
+                                               YLength=num_wl_bins)
+        # TODO get the range from the reduction properties
+        xaxis = np.arange(0., float(num_wl_bins + 1)) * (2.20 - 0.05) / (num_wl_bins) + 0.05
```
Comment (marshallmcdonnell) [inline Testing/SystemTests/tests/analysis/AlignAndFocusPowderFromFilesTest.py]: Just to check, 2.20 is WavelengthMax? Should that be a variable for clarity?

**Review by marshallmcdonnell — CHANGES_REQUESTED** [review]
Overall, great addition for system tests! Few comments above need to be addressed.

**Review by marshallmcdonnell — APPROVED** [review]

**Comment by martyngigg** [comment]
I'm just going to check that the new tests pass on Windows/macOS before saying yes.

**Comment by martyngigg** [comment]
Tests pass on macOS.

**Comment by marshallmcdonnell** [comment]
FYI, I test on Ubuntu 18.04 for the review

**Comment by martyngigg** [comment]
Thanks for confirming @marshallmcdonnell. We've had a few tests go through recently that have broken on Windows/macOS so I just wanted to be doubly sure before proceeding, especially as this had a test that claimed it used to only work on RHEL 6!

**Comment by peterfpeterson** [comment]
The `_skip_test` function was unused and deleted in this PR. It was a copy/paste mistake from `SNSPowderRedux` which this was based on.

**Comment by martyngigg** [comment]
@SamJenkins1 Can you provide some more information? Console output?

**Pull request merged**